doc/manual/manual.tex

```diff
@@ -1138,6 +1138,46 @@ \subsection{Compositional fields}
 temperature set of equations. Play with them!
 
 
+\subsection{Constitutive laws}
+
+Equation (\ref{eq:stokes-1}) describes buoyancy-driven flow in an isotropic
+fluid where strain rate is related to stress by a scalar field, $\eta$. For
```

Comment:
I think it would be clearer to say "by a scalar (possibly spatially variable) multiplier, $\eta$".